how-to/privileged.md

```diff
@@ -0,0 +1,45 @@
+# Privileged Kata Containers
+
+Kata Containers supports creation of containers that are "privileged" - i.e. have additional capabilities and access
+that isn't normally granted.
+
+* [Caveats](#caveats)
+    * [Host Devices](#host-devices)
+        * [Containerd and CRI](#containerd-and-cri)
+
+## Caveats
+
+Whilst this functionality is supported, it can decrease the security of Kata Containers if not configured correctly.
```

Comment:
Could you make this a warning admonition using the formatting explained in:

https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#warnings-and-other-admonitions

... so that it renders as:

> **Warning:** Whilst this functionality is supported, it can decrease the security of Kata Containers if not configured correctly.